requirements.txt

```diff
@@ -32,4 +31,5 @@ packaging>=20.4
 wheel>=0.38.0
 # not directly required, pinned by Snyk to avoid a vulnerability
 pygments>=2.7.4
+--extra-index-url https://pypi.ngc.nvidia.com
```

Comment:
This one still fails in circleci because it reads requirements line-by-line to get rid of the comments
We have 
pip install --extra-index-url https://pypi.ngc.nvidia.com
pip install pytorch-quantization==2.1.2

Reply:
Why do we need comments in requirements.txt?

Reply:
Why do we need a special way of reading requirements.txt if comments are the part of PEP?